project/src/main/java/com/google/sps/servlets/TokenValidator.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.import java.io.IOException;
+
+package com.google.sps.servlets;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.UserVerifier;
+import java.util.Optional;
+
+/**
+ * A utility class for validating user tokens.
+ */
+public final class TokenValidator {
+
+  /**
```

Comment:
Please add a sentence explaining what this function does. Make sure to state that this function modifies the response in case of invalid token ID.

Reply:
Good point, done